### ISSUE TITLE
Request: more details on obtaining credentials

### ISSUE THREAD

**Issue by simonw** [domenp/aircal#4]
The README currently says:

> Google API credentials are required to create events in the calendar. You can obtain them [here](https://console.developers.google.com/apis/credentials). Store `credentials.json` into a directory accessible by your code.

I couldn't figure out how to use the linked interface to obtain `credentials.json` file that I could use with this project.

**Comment by domenp** [comment]
You need to obtain OAuth 2.0 client credentials from the Google API Console. More [here](https://developers.google.com/identity/protocols/oauth2).

Does that help?

**Comment by simonw** [comment]
Not really. I've been working with OAuth for 13 years and after 15 minutes of messing around in the console I still hadn't figured out how to get a calendar credential I could use with this tool!

Step by step instructions would be really useful. Google's documentation around this is not good enough, unfortunately.

**Comment by simonw** [comment]
I got [a tip](https://twitter.com/mheap/status/1321845437335261184) on Twitter that pointed me to https://michaelheap.com/google-oauth-with-node-js/ which looks like it might have enough information for me to get a token.

**Comment by domenp** [comment]
Great, let me know if the linked instructions worked for you?

**Comment by simonw** [comment]
I'll try that today. Do you know if it's possible to get credentials without creating a new OAuth application?

**Comment by domenp** [comment]
Don't know to be honest.

Believe the easiest way is to go via "Create credentials -> OAuth Client ID" workflow in Google API console. This does create a new OAuth app, I believe.
